# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../poppler/poppler-0.5.4-fontconfig.patch
# Copyright (C) 2006 - 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Problem: poppler doesn't inject fontconfig CFLAGS and LIBS
Bug-Report: https://bugs.freedesktop.org/show_bug.cgi?id=8415

[Note]
The patch was send upstream and one of the poppler maintainers said that he will
fix the problem within the next days. ( 2006-09-25 morfoh)

diff -ruN poppler-0.5.4-orig/glib/Makefile.am poppler-0.5.4/glib/Makefile.am
--- poppler-0.5.4-orig/glib/Makefile.am	2006-05-20 00:09:00.000000000 +0200
+++ poppler-0.5.4/glib/Makefile.am	2006-09-25 14:56:24.000000000 +0200
@@ -17,6 +17,7 @@
 	$(cairo_includes)			\
 	$(POPPLER_GLIB_CFLAGS)			\
 	$(FREETYPE_CFLAGS)			\
+	$(FONTCONFIG_CFLAGS)                    \
 	$(cairo_includes)			
 
 stamp_files = stamp-poppler-enums.h
@@ -84,6 +85,7 @@
 	$(top_builddir)/poppler/libpoppler.la		\
 	$(POPPLER_GLIB_LIBS)				\
 	$(FREETYPE_LIBS)				\
+	$(FONTCONFIG_LIBS)				\
 	$(cairo_libs)
 
 libpoppler_glib_la_LDFLAGS = -version-info 1:0:0
diff -ruN poppler-0.5.4-orig/qt/Makefile.am poppler-0.5.4/qt/Makefile.am
--- poppler-0.5.4-orig/qt/Makefile.am	2006-07-24 21:04:50.000000000 +0200
+++ poppler-0.5.4/qt/Makefile.am	2006-09-25 14:57:22.000000000 +0200
@@ -4,7 +4,8 @@
 	$(cairo_includes)			\
 	$(POPPLER_QT_CXXFLAGS)			\
 	$(SPLASH_CFLAGS)			\
-	$(FREETYPE_CFLAGS)			
+	$(FREETYPE_CFLAGS)			\
+	$(FONTCONFIG_CFLAGS)
 
 
 poppler_includedir = $(includedir)/poppler
@@ -26,7 +27,8 @@
 libpoppler_qt_la_LIBADD = 			\
 	$(top_builddir)/poppler/libpoppler.la	\
 	$(POPPLER_QT_LIBS)			\
-	$(FREETYPE_LIBS)
+	$(FREETYPE_LIBS)			\
+	$(FONTCONFIG_LIBS)
 
 libpoppler_qt_la_LDFLAGS = -version-info 1:0:0
 
diff -ruN poppler-0.5.4-orig/test/Makefile.am poppler-0.5.4/test/Makefile.am
--- poppler-0.5.4-orig/test/Makefile.am	2006-04-12 00:23:28.000000000 +0200
+++ poppler-0.5.4/test/Makefile.am	2006-09-25 14:59:37.000000000 +0200
@@ -33,7 +33,8 @@
 	-I$(top_srcdir)/glib			\
 	-I$(top_builddir)/glib			\
 	$(cairo_includes)			\
-	$(GTK_TEST_CFLAGS)			
+	$(GTK_TEST_CFLAGS)			\
+	$(FONTCONFIG_CFLAGS)
 
 noinst_PROGRAMS = $(gtk_splash_test) $(gtk_cairo_test) $(pdf_inspector)
 
@@ -51,7 +52,8 @@
 	$(top_builddir)/poppler/libpoppler.la		\
 	$(top_builddir)/glib/libpoppler-glib.la		\
 	$(CAIRO_LIBS)					\
-	$(GTK_TEST_LIBS)
+	$(GTK_TEST_LIBS)				\
+	$(FONTCONFIG_LIBS)
 
 pdf_inspector_SOURCES =			\
        pdf-inspector.cc
diff -ruN poppler-0.5.4-orig/utils/Makefile.am poppler-0.5.4/utils/Makefile.am
--- poppler-0.5.4-orig/utils/Makefile.am	2006-04-12 00:25:41.000000000 +0200
+++ poppler-0.5.4/utils/Makefile.am	2006-09-25 15:00:46.000000000 +0200
@@ -14,11 +14,13 @@
 	-I$(top_srcdir)				\
 	-I$(top_srcdir)/utils			\
 	-I$(top_srcdir)/poppler			\
-	$(UTILS_CFLAGS)
+	$(UTILS_CFLAGS)				\
+	$(FONTCONFIG_CFLAGS)
 
 LDADD =						\
 	$(top_builddir)/poppler/libpoppler.la	\
-	$(UTILS_LIBS)
+	$(UTILS_LIBS)				\
+	$(FONTCONFIG_LIBS)
 
 bin_PROGRAMS =					\
 	pdffonts				\
